### DISCUSSION TITLE
Duda ejercicio QMP1 - Jueves Mañana

### DISCUSSION THREAD

**Discussion by antobevi** [dds-jv/foro#183] [Consultas generales] [answered]
Buenas profe, no me quedó muy claro que es lo que hay que hacer para el ejercicio de Qué me pongo. Tenemos que hacer el Diagrama de Clases, el pseudocódigo y además bajar eso a código, o solamente el Diagrama de Clases y el pseudocódigo?
Gracias!

**Comment by gastonprieto** [comment, marked as answer]
Buenas @antobevi !  

El de QMP es como los que hicimos la clase pasada, la idea es practicar la comunicación de una solución, esto incluye prosa (si es que queres aclarar algo), pseudocódigo para entender la implementación y el diagrama de clases para tener una visión general del sistema.

No es necesario que lo bajes a código Java, si lo queres a practicar no hay drama, pero no es obligatorio.

**Reply by antobevi** [reply]
Gracias profe!
